app.json

```diff
@@ -46,6 +45,18 @@
       "description": "This ensures the Super Scaffolding templates don't show up in production.",
       "generator": "secret",
       "required": true
+    },
+    "RAILS_MAX_THREADS": {
+      "description": "The maximum number of threads that puma will fork. https://devcenter.heroku.com/articles/deploying-rails-applications-with-the-puma-web-server#recommended-default-puma-process-and-thread-configuration",
+      "value": "5"
```

Comment:
Rails 8 is going to be shifting to a default of 3, I think we should suggest the same despite Heroku's defaults. I can't remember if they did a more condensed write up of it, but there was quite a long thread of discussion on GitHub.

Reply:
@kaspth I'd be interested in learning more about that discussion. I also tested this number with the demo site and it's working well and keeping things withing the memory quota. What benefits do you see with defaulting to 3 instead of 5?

Reply:
Gonna go ahead and merge this as-is for now. We can always circle back later and change it if we need to.